### PR TITLE
Target update

### PR DESCRIPTION
**Pull request by userhas404d**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -159,15 +159,15 @@ docs/generate/%:
 	mv -f $(README_TMP) $*
 	@ echo "[$@]: Documentation files creation complete!"
 
-docs/lint/%:
+docs/lint/%: | terraform/lint
```
Comment (lorengordon) [inline Makefile]: Did you mean to put this on the /% target?

Comment (userhas404d) [reply Makefile]: negatory, updated

File: tests/make/docs_generate_success.bats [hunk 2]
```diff
@@ -4,20 +4,27 @@ TEST_DIR="$(pwd)/docs_generate_success"
 
 # generate a test terraform project with a nested "module"
 function setup() {
+rm -rf "$TEST_DIR"
```
Comment (lorengordon) [inline tests/make/docs_generate_success.bats]: somewhat curious on these changes... is `teardown()` not doing its job? `cat >` should also make `touch` unnecessary, right?

Comment (userhas404d) [reply tests/make/docs_generate_success.bats]: re`touch`: I was running into some weird errors while testing things and just threw these in there without digging too deep. 

The `rm` line was from when I was going back and forth between commenting out the `teardown()` function while getting the test(s) to pass. it was useful enough and low impact enough that I just left it

Comment (lorengordon) [reply tests/make/docs_generate_success.bats]: sounds like `bats` could use a debug mode, where it steps through its functions one-by-one so you can inspect the state and then continue...

**Pull request merged**